Test/logger/ProofDependencyWarnings.dfy.expect

```diff
@@ -0,0 +1,20 @@
+ProofDependencyLogging.dfy(176,11): Warning: unnecessary assumption
+ProofDependencyLogging.dfy(186,13): Warning: vacuous proof: assertion always holds
```

Comment:
Ah NOW I get my previous confusion - in this specific (and common) case, the `vacuous proof: <proof description>` template leads to a string that's too easy to misinterpret as `vacuous proof, or, as another way of saying 'vacuous proof', assertion always holds`. :)

Perhaps we could use a template like `vacuous proof of (<proof description>)` instead?

Reply:
We could also consider not using "vacuous" since many users won't be familiar with that concept. Perhaps "Warning: proved from impossible assumptions: ..." ("contradictory assumptions" is appealing too, but then `false` by itself is impossible).

Reply:
Yeah, so how about "proved using contradictory assumptions: <proof description>"? I could change the name of `--warn-vacuity`, too. Maybe `--warn-contradictory-assumptions`? That's nicely symmetric with `--warn-redundant-assumptions`.